src/app/onion/core/learning-object.service.ts

```diff
@@ -195,12 +194,13 @@ export class LearningObjectService {
     id: string,
     authorUsername: string,
     learningObject: { [key: string]: any },
+    reason?: string,
```

Comment:
Can you just enlighten me why reason is an optional parameter on these routes.

Reply:
Reason is only needed when you are moving to the accepted major/minor and rejected statuses.  All other statuses don't need them.